presto-main/src/main/java/com/facebook/presto/metadata/OperatorType.java

```diff
@@ -201,6 +201,11 @@ public String getOperator()
 
     abstract void validateSignature(TypeSignature returnType, List<TypeSignature> argumentTypes);
 
+    public static void validateOperator(OperatorType operatorType, TypeSignature returnType, List<TypeSignature> argumentTypes)
```

Comment:
I tried to make the abstract method above public, but that requires changing all the implementations, which I avoided since they get removed on the next commit. So this commit just throws in this dumb accessor method.